pages/api/graphql-rest.page.ts

```diff
@@ -328,16 +328,20 @@ class MpdxRestApi extends RESTDataSource {
 
   async getCoachingAnswerSets(
     accountListId: string,
+    organizationId: string | null | undefined,
     completed: boolean | null | undefined,
     multiple: boolean,
   ) {
-    const response = await this.get(
-      `coaching/answer_sets?filter[account_list_id]=${accountListId}&filter[completed]=${
-        completed ?? false
-      }${
-        multiple ? '' : '&filter[limit]=1'
-      }&include=answers,questions&sort=-completed_at`,
-    );
+    const response = await this.get(`coaching/answer_sets`, {
+      params: {
+        'filter[account_list_id]': accountListId,
+        'filter[organization_id]': organizationId ?? undefined,
```

Comment:
If the `organizationId` isn't set because the user doesn't have a `salaryOrganizationId` set on their account list, we fall back to loading all of them, just like we used to.